lsif-java/src/main/scala/com/sourcegraph/lsif_java/SemanticdbPrinters.scala

```diff
@@ -17,7 +17,7 @@ object SemanticdbPrinters {
       .linesWithSeparators
       .zipWithIndex
       .foreach { case (line, i) =>
-        out.append(line)
+        out.append(line.replace("\t", "→"))
```

Comment:
Im curious if theres a reason we cant do `replace("\t", "    ")` instead here.

Reply:
I we can do that but it removes the fact that there's a tab character, which may be helpful to debug an issue.